optuna/storages/_in_memory.py

```diff
@@ -287,7 +287,7 @@ def get_best_trial(self, study_id: int) -> FrozenTrial:
             if best_trial_id is None:
                 raise ValueError("No trials are completed yet.")
             elif len(self._studies[study_id].directions) > 1:
-                raise ValueError(
```

Comment:
Although `BaseStorage` says `RuntimeError` is raised when `get_best_trial` is called for the multi-objective study, it actually raises `ValueError`. I fixed the mismatch even this PR only aims to improve the coverage.

https://github.com/optuna/optuna/blob/c9ea7abc081253e01c54b15d7019bbbeac4c97d6/optuna/storages/_base.py#L607-L608

Reply:
Update: this fix is extracted to https://github.com/optuna/optuna/issues/3491.
Reverted in [41f82cd](https://github.com/optuna/optuna/pull/3480/commits/41f82cd92d7ccffe1637b30c79ac59596311afe9).